homeassistant/components/template/light.py

```diff
@@ -1,13 +1,18 @@
 """Support for Template lights."""
 import logging
 
+from ast import literal_eval as make_tuple
```

Comment:
We should not use this. Also renaming it `make_tupel`, it's actually a dangerous method, especially since it is run on unknown data.

Reply:
Oh I was not aware about this. Thx.